python/samples/04.ai.b.messageExtensions.AI-ME/env/.env.dev.user

```diff
@@ -1,8 +1,3 @@
 # This file includes environment variables that will not be committed to git by default. You can set these environment variables in your CI/CD system for your project.
 
-# Secrets. Keys prefixed with `SECRET_` will be masked in Teams Toolkit logs.
```

Comment:
Can you share some background about removing the content? The purpose to having these content is to let users know what will be generated during provision / local debug, as well as letting them easily fill an existing value if they want.

The Azure OpenAI envs is also required when provision an Azure environment.

Reply:
I was thinking we can include this comment in the READMEs instead, but open to keeping it here as well. 

AOAI isn't supported right now (moderation logic requires fixes in the `openai-python` library) so I didn't include it here so as to avoid confusion

Reply:
Thank you. Let keep the comments in .env files first to align with TTK templates and we can discuss further if needed.

The AOAI/OAI envs in `.env.dev.user` are used to configure Azure App Service app settings. TTK does not consume information from `.env` file in this step as it may contain envs that not related to the `dev` environment. So we still need to add a placeholder to `.env.dev.user` for users to easily provide AOAI/OAI envs for the dev environment.

For AOAI/OAI envs used for local debugging, I think it's already aligned with JS/C# samples by haveing a `sample.env` in the root.